android/app/build.gradle

```diff
@@ -31,6 +31,11 @@ android {
         ndk {
             abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
         }
+
+        // This will later be used to control the test apk build type
+        testBuildType System.getProperty('testBuildType', 'debug')
```

Comment:
Where is this used? I see it in the build command line, is that all? It's also se to the same value.

Reply:
Yes. That's the only place where it's used.